samples/wifi/scan/src/main.c

```diff
@@ -332,8 +333,16 @@ int main(void)
 			}
 		}
 
+		ret = net_bytes_from_str(params.mac_address.addr, sizeof(CONFIG_WIFI_MAC_ADDRESS),
+					 CONFIG_WIFI_MAC_ADDRESS);
+		if (ret) {
+			LOG_ERR("Failed to parse MAC address: %s (%d)",
+						CONFIG_WIFI_MAC_ADDRESS, ret);
```

Comment:
Looks like you are still using tabs, typically we align to open brace with tabs + spaces but without using too many tabs